src/components/navigation/SideMenu.tsx

```diff
@@ -90,7 +91,11 @@ const VersionInfo = styled.div`
     }
 `;
 
-const SideMenu = (): JSX.Element => {
+const SideMenu = ({
+    isOffline = false,
+}: {
+    isOffline?: boolean;
```

Comment:
Kan du lage et interface for dette heller? interface SideMenuProps. Jeg vet det bare er en verdi, men det er slik vi gjør det i resten av kodebasen :)